src/overlays/actors/ovl_En_Rz/z_en_rz.c

```diff
@@ -184,16 +191,29 @@ void EnRz_ActorShadowFunc(Actor* thisx, Lights* mapper, PlayState* play) {
 }
 
 void EnRz_ChangeAnim(PlayState* play, EnRz* this, s16 animIndex, u8 animMode, f32 transitionRate) {
+    static AnimationHeader* D_80BFCD20[] = {
+        &object_rz_Anim_00457C, &object_rz_Anim_003A20, &object_rz_Anim_005E50, &object_rz_Anim_003098,
+        &object_rz_Anim_00059C, &object_rz_Anim_000DE8, &object_rz_Anim_0028D4,
+    };
+
+    static AnimationHeader* D_80BFCD3C[] = {
+        &object_rz_Anim_003A20, &object_rz_Anim_003A20, &object_rz_Anim_005390, &object_rz_Anim_003098,
+        &object_rz_Anim_00059C, &object_rz_Anim_000DE8, &object_rz_Anim_0028D4,
+    };
+
+    static LinkAnimationHeader* sLinkAnimations[] = { &gPlayerAnim_link_normal_wait_free,
+                                                      &gPlayerAnim_alink_dance_loop };
     f32 endFrame;
     AnimationHeader** animationPtr;
 
-    if (!EN_RZ_GET_PARAM_8000(&this->actor)) {
+    if (!EN_RZ_GET_SISTER(&this->actor)) {
         animationPtr = D_80BFCD20;
     } else {
         animationPtr = D_80BFCD3C;
```

Comment:
So one of these arrays is for Judo, the other for Marilla and we can name them as such.